packages/server/lib/modes/run.js

```diff
@@ -1257,14 +1297,22 @@ module.exports = {
         }
       }
 
-      if (testingType === 'e2e') {
-        // always close the browser now as opposed to letting
-        // it exit naturally with the parent process due to
-        // electron bug in windows
+      // Close the browser if the environment variable is set to do so
+      if (process.env.CYPRESS_INTERNAL_FORCE_BROWSER_RELAUNCH) {
         debug('attempting to close the browser')
         await openProject.closeBrowser()
+      } else {
+        // Otherwise, reset the browser and close the browser tabs
+        debug('stopping screencast and resetting browser state')
+        await Promise.all([openProject.stopScreencast(), openProject.resetBrowserState()])
```

Comment:
I see the problem now. We likely cannot reset the browser state with a live AUT. We may first want to kill the AUT (by removing the iframe first in the driver code) so we can safely reset the browser state. Or come up with a slightly different approach regarding stopping the video if that's broken by closing the browser tab.